app/controllers/application_controller.rb

```diff
@@ -19,4 +21,18 @@ def admin_authenticated
   def require_user
     render file: '/public/404' unless current_user
   end
+
+  def store_location
+    return unless request.get?
```

Comment:
I know you haven't pushed to Heroku but wanted to add some notes on my experience as a user:

I clicked on the Login using Google link and was able to put in my password and go through the OAuth experience. However, it redirected me to the standard Heroku 404 page.  When I navigated back to the landing page I noticed that the OAuth had worked because my google name was visible. Thought that was interesting!